src/analyzer/protocol/rawstream/events.bif

```diff
@@ -0,0 +1,27 @@
+## Generated for every DeliverStream() invocation on an instance of the Raw analyzer.
+##
+## This is a low-level event to inspect all data from the originator
+## and responder for debugging or logging of plain-text interactive
+## sessions.
```

Comment:
I'd add a warning that it's potentially expensive if substantial amounts of data end up going through scriptland.